src/app/account/PageActivate.tsx

```diff
@@ -17,7 +17,7 @@ export const PageActivate = () => {
   const { searchParams } = useSearchParams();
 
   useEffect(() => {
-    activateAccount({ key: searchParams.get('key') });
+    activateAccount({ key: searchParams.get('key') ?? 'KEY_NOT_DEFINED' });
```

Comment:
Shouldn't we put this string as a constant somewhere ? Or this may be overkill.

Reply:
We could, but the point of this string is for the developer to see it in the URL if something wrong is happening, by putting directly the value here, the developer can `CTRL+F` and find directly its usage.

Reply:
But we could put it in a const that is near the usage.

Reply:
If this intent is to help the dev then putting it here is the best I think